web/src/beta/components/Icon/Icons/appearance.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M6.0238 7.27979L2.35728 13.8884C2.29861 13.9942 2.26768 14.1142 2.26758 14.2364C2.26748 14.3585 2.29823 14.4786 2.35673 14.5844C2.41523 14.6903 2.49942 14.7782 2.60083 14.8393C2.70224 14.9004 2.81729 14.9326 2.9344 14.9326H10.2674C10.3845 14.9326 10.4996 14.9004 10.601 14.8393C10.7024 14.7782 10.7866 14.6903 10.8451 14.5844C10.9036 14.4786 10.9343 14.3585 10.9342 14.2364C10.9341 14.1142 10.9032 13.9942 10.8445 13.8884L7.17802 7.27979C7.11945 7.17422 7.03529 7.08656 6.934 7.02563C6.83271 6.96469 6.71784 6.93262 6.60091 6.93262C6.48398 6.93262 6.36911 6.96469 6.26782 7.02563C6.16653 7.08656 6.08237 7.17422 6.0238 7.27979V7.27979Z" stroke="#E0E0E0" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
When you add new icons, please make all stroke/fills `currentColor` so we can update the colors through the Icon component

Reply:
Stroke